horace_core/utilities/@d2d/private/calculate_coord_change.m

```diff
@@ -69,13 +69,13 @@
 %wrong!
 ax(isnan(ax))=1;
 if any(abs(ax)>1.001)
-    error('Horace error: Brillouin zone centres are not equivalent'); 
+    error('Horace error: Brillouin zone centres are not equivalent');
 end
 
 %Now we work out how to alter each of the objects:
 %
-coords1=w1.data.pix([1:3],:);
-coords2=w2.data.pix([1:3],:);
+coords1=w1.data.pix.q_coordinates;
```

Comment:
are these local variables useful any longer? You could use `wN.data.pix.q_coordinates` directly in the only place each of these are used.

Reply:
is `fullax=repmat(ax,1,(numel(coords1))/3);` on line 95 below a ca;; to w1.data.pix.num_pixels?